src/stores/NotificationsController.ts

```diff
@@ -14,30 +15,36 @@ export enum NotificationsTypes {
 }
 
 export const subscribeToNotifications = async (uuid: string = '') => {
-  return await useService(async ({ accountService }) => {
-    return await accountService.subscribeToNotifications(uuid, async (message) => {
-      if (message) {
-        const notification = JSON.parse(message);
-        switch (notification.type) {
-          case NotificationsTypes.FAILED:
-          case NotificationsTypes.SUBSCRIBE_FAILED:
-            window.location.href = addQueryParams(window.location.href, { u: 'payment-error', message: notification.resource?.message });
-            break;
-          case NotificationsTypes.ACCESS_GRANTED:
-            await reloadActiveSubscription();
-            window.location.href = addQueryParams(window.location.href, { u: 'welcome' });
-            break;
-          case NotificationsTypes.ACCESS_REVOKED:
-            await reloadActiveSubscription();
-            break;
-          case NotificationsTypes.CARD_REQUIRES_ACTION:
-          case NotificationsTypes.SUBSCRIBE_REQUIRES_ACTION:
-            window.location.href = notification.resource?.redirect_to_url;
-            break;
-          default:
-            break;
+  return await useAccount(async () => {
+    return await useService(async ({ accountService }) => {
+      return await accountService.subscribeToNotifications(uuid, async (message) => {
+        if (message) {
+          const notification = JSON.parse(message);
+          switch (notification.type) {
+            case NotificationsTypes.FAILED:
+            case NotificationsTypes.SUBSCRIBE_FAILED:
+              window.location.href = addQueryParams(window.location.href, { u: 'payment-error', message: notification.resource?.message });
+              break;
+            case NotificationsTypes.ACCOUNT_LOGOUT:
+              await getAccount();
+              window.location.href = addQueryParams(window.location.href, { u: 'simultaneous-logins' });
```

Comment:
we need to show SimultaneousLoginModal warning only if the user reached the limit of simultaneous logins, not on every logout.